sdk/storage/sample/CMakeLists.txt

```diff
@@ -1,13 +1,13 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # SPDX-License-Identifier: MIT
 
-cmake_minimum_required (VERSION 3.12)
+cmake_minimum_required (VERSION 3.15)
 
 if (BUILD_CURL_TRANSPORT)
 
 add_executable (
      azure-storage-sample
-     blob_getting_started.cpp
+     datalake_getting_started.cpp
```

Comment:
So you replace the blob sample with datalake sample. Where the blob sample should go?

Reply:
There are multiple main functions here, and that is what I want to get rid of. Added blob_getting_started.cpp back and commented out.

Reply:
No, you cannot just remove blob sample. I mean blob sample is as important as datalake sample.

we have to think of a way to make two or more samples exist together.

Reply:
Yes, like the current approach in track 1. I'll do some more investigation on it but it's hardly a priority for now.

Reply:
Added an issue to track: https://github.com/Azure/azure-sdk-for-cpp/issues/193